packages/frontend/components/Video.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from '../styles/videoBoard.module.scss'
+
+export interface VideoInterface {
+  id: string
+  title: string
+  creator: string
+  url: string
+}
+
+interface VideoProps {
+  key: string
+  video: VideoInterface
+}
+
+export default function Video(props: VideoProps) {
+  return (
+    <div>
+      <h3 className={styles.cardInfo}>{props.video.title}</h3>
+      <h4 className={styles.cardInfo}> - {props.video.creator}</h4>
```

Comment:
these should be the same size, and always use a p if you style these externally. The creator should be cursive like in the design.
```suggestion
      <p className={styles.cardInfo}>{props.video.title}</p>
      <p className={styles.cardInfo}> - {props.video.creator}</p>
```